app/llm-test.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+from langchain_openai import ChatOpenAI
+from langchain_core.prompts import ChatPromptTemplate
+from langchain_core.output_parsers import StrOutputParser
+from langchain_core.messages import HumanMessage, AIMessage
+from langchain_openai import AzureChatOpenAI
+
+os.environ["AZURE_OPENAI_API_KEY"] = "f294412f93f348e6a8d839e196456a2e"

```

Comment:
We should not check in the keys. They should be read from the `.env` file or environemnt.